tests/importer/onnx_/basic/test_reduce.py

```diff
@@ -120,11 +121,12 @@ def _make_module(in_shape, in_datatype, reduce_op, axes, keepdims, op_version):
 @pytest.mark.parametrize('op_version', op_version_lists)
 def test_reduce(in_shape, in_datatype, reduce_op, axes, keepdims, request, op_version):
     if len(axes) <= len(in_shape):
-        model_def = _make_module(in_shape, in_datatype, reduce_op, axes, keepdims, op_version)
+        if reduce_op == 'ReduceSum' and op_version < 13:
```

Comment:
this condition only test ReduceSum and op_version 13.

Reply:
fixed